snap/snapcraft.yaml

```diff
@@ -676,18 +720,36 @@ parts:
   edgex-go:
     source: .
     plugin: make
-    after: [go]
+    after: [go113]
     override-build: |
+      export PATH="$SNAPCRAFT_STAGE/go1.13/bin:$GOPATH/bin:$PATH"
       cd $SNAPCRAFT_PART_SRC
+
       make build
 
       # copy service binaries, configuration, and license files into the snap install
       for service in core-command core-data core-metadata support-logging support-notifications support-scheduler sys-mgmt-agent \
-          security-proxy-setup security-secrets-setup security-secretstore-setup security-file-token-provider; do
+          security-proxy-setup security-secrets-setup security-secretstore-setup security-file-token-provider security-secretstore-read; do
 
           install -DT "./cmd/$service/$service" "$SNAPCRAFT_PART_INSTALL/bin/$service"
 
-          if [ "$service" != "security-file-token-provider" ]; then
+          if [ "$service" = "security-secretstore-read" ]; then
+              mkdir -p "$SNAPCRAFT_PART_INSTALL/config/$service/res/"
+
+              # since security-secretstore-read doesn't support config env overrides, edit the
+              # configuration file at build time
+              CONFIG_FILE="./cmd/$service/res/configuration.toml"
+              if grep -q "/tmp/edgex" "$CONFIG_FILE"; then
```

Comment:
Thanks for moving this here

Reply:
What was the ping for?

Reply:
you missed a comment from my previous review about why we are hard-coding `VAULT_SKIP_VERIFY: "true"`